### DISCUSSION TITLE
Release 1.5.3

### DISCUSSION THREAD

**Discussion by BRU-EUSurvey** [EUSurvey/EUSURVEY#528] [Announcements]
**ABOUT THIS RELEASE**
This release encompasses various functional changes and improvements. It is also a major system upgrade.

**Main changes**
SCQ = Single Choice Question
MCQ = Multiple Choice Question
OSS = Open Source Software

- Accessing survey results lead now to the creation of a “Results page was accessed” entry in the Activity Log. This is for a better tracking of who access the results.
- A new button style has been added for SCQ. It displays the SCQ Answers in the form of a button group.
- An 'Exclusive' option is now added to Answer elements for MCQ. If used, then when the participant ticks the corresponding answer, all the other answers are automatically unticked. This is to cover the “none of the above” use-case.
- Missing translations for web labels have been added for all 24 EU languages.
- OSS Version: We have removed from the survey creation dialog the confirmation message regarding high volume of expected replies: “I confirm that we will inform the EUSurvey Team if we expect a very high number of responses to this survey…”.
- A filter on the Type of survey (e.g. standard, quiz) has been added in the Surveys tab.
- We have changed the message from “too many selection” to “maximum X selection(s)” for the MCQ element? This is to make it clearer for the survey respondent.
- For the eVote module, counting of votes are now logged in details in the Activity Log (access to the quorum, display results, allocate seats).
- Web accessibility for the Get PDF dialog box has been improved. 
- Data Protection Agreement: Our DPA is now communicated through a link in the FAQ (new article) as well as in the survey creation dialog for external users.
- The Survey Title is now included in the dialog to copy a survey. This is to make the copied survey more distinguishable from the original one.
- Addition of an important FAQ article regarding the protection of data (right to access, modify, delete etc.). This helps the form manager to know what to do in case of data protection request.
- We have added a new FAQ article to explain the “Formula field” question type. 
- Delphi: Generated PDF report can now include the graphics.
- We have implemented two new API methods: getDeletedContributions & getPrivilegedUsers. More information available in the API specifications document. ModifiedDate has been added to the web service getSurveyMetadata 
- Quiz timer has been improved with a date picker.
- Addition of a new FAQ article to explain the Complex Table.
- Some old skins have been removed to avoid the user to use them.

**Comment by christceder** [comment]
> **OM DENNA RELEASE** Den här utgåvan omfattar olika funktionella förändringar och förbättringar. Det är också en stor systemuppgradering.
> 
> **Huvudändringar** SCQ = Single Choice Question MCQ = Flervalsfråga OSS = Open Source Software
> 
> * Åtkomst av enkätresultat leder nu till skapandet av en "Resultatsida öppnades" i aktivitetsloggen. Detta för att bättre kunna spåra vem som kommer åt resultaten.
> * En ny knappstil har lagts till för SCQ. Den visar SCQ Answers i form av en knappgrupp.
> * Ett "Exklusivt" alternativ har nu lagts till i Answer elements for MCQ. Om det används, när deltagaren kryssar i motsvarande svar, avmarkeras alla andra svar automatiskt. Detta för att täcka användningsfallet "inget av ovanstående".
> * Saknade översättningar för webbetiketter har lagts till för alla 24 EU-språk.
> * OSS-version: Vi har tagit bort från dialogrutan för att skapa enkäten bekräftelsemeddelandet angående hög volym förväntade svar: "Jag bekräftar att vi kommer att informera EUSurvey Team om vi förväntar oss ett mycket stort antal svar på denna undersökning...".
> * Ett filter på Typ av undersökning (t.ex. standard, frågesport) har lagts till på fliken Undersökningar.
> * Vi har ändrat meddelandet från "för många val" till "maximala X val" för MCQ-elementet? Detta för att göra det tydligare för respondenten.
> * För eVote-modulen loggas nu rösträkningsdetaljer i aktivitetsloggen (tillgång till beslutförhet, visa resultat, fördela platser).
> * Webbtillgängligheten för dialogrutan Hämta PDF har förbättrats.
> * Dataskyddsavtal: Vårt dataskyddsavtal kommuniceras nu via en länk i FAQ (ny artikel) samt i dialogrutan för att skapa enkäter för externa användare.
> * Enkättiteln finns nu med i dialogrutan för att kopiera en undersökning. Detta för att göra den kopierade enkäten mer särskiljbar från den ursprungliga.
> * Tillägg av en viktig FAQ-artikel om skydd av data (rätt till åtkomst, ändring, radering etc.). Detta hjälper formulärhanteraren att veta vad den ska göra vid begäran om dataskydd.
> * Vi har lagt till en ny FAQ-artikel för att förklara frågetypen "Formelfält".
> * Delphi: Genererad PDF-rapport kan nu inkludera grafiken.
> * Vi har implementerat två nya API-metoder: getDeletedContributions och getPrivilegedUsers. Mer information finns i API-specifikationsdokumentet. ModifiedDate har lagts till i webbtjänsten getSurveyMetadata
> * Quiztimern har förbättrats med en datumväljare.
> * Tillägg av en ny FAQ-artikel för att förklara den komplexa tabellen.
> * Vissa gamla skal har tagits bort för att undvika att användaren ska använda dem.

**Comment by Gchirico63** [comment]
I am new to EUsurvey. I am preparing a survey to be shared within a Cost Action. 
ISSUE: 
I cannot find the way to add colleagues to the existing survey and give them the privileges to edit the Survey or at least to have direct access to the results of the survey. 

the instructions found online do not work for this. 

the error on "adding a user" is that the email is not found, even though it is in my address book.  

thank you
giberto

**Reply by origabe** [reply]
Dear,
Are you using the service available here: https://ec.europa.eu/eusurvey ?
If yes, then the person you would like to give access to the survey has to create an EULogin account first: they can create one here: https://webgate.ec.europa.eu/cas/eim/external/register.cgi
Once the account created you will be able to find the user in the 'Add privileges' page, mentioning 'External' as a domain.
KR,
Benoît

**Comment by Gchirico63** [comment]
thank you for answering my question.
however, this does not fix the problem.
I do access EUsurvey from that link and I have emails of colleagues that
are the ones linked to their EUcordis account (i.e. the account on
which each
of us access to the EU project).

However, as you see from the dump of the screens I have not the possibility
to specify that the user is "External" when I go the "privileges" page.
Nor seems to me that I have this possibility when I add the user to my
addressbook (I am not sure about this last thing, I need to check).

I would really appreciate if you could help me in this task. It is a survey
for a COST action and I would like that  it is accessible to the whole core
group but this is created under my personal EU account.
thank you
kind regards
Giuseppe Chirico

[image: Dumps.jpg]


Il giorno ven 17 mag 2024 alle ore 07:58 origabe ***@***.***>
ha scritto:

> Dear,
> Are you using the service available here: https://ec.europa.eu/eusurvey ?
> If yes, then the person you would like to give access to the survey has to
> create an EULogin account first: they can create one here:
> https://webgate.ec.europa.eu/cas/eim/external/register.cgi
> Once the account created you will be able to find the user in the 'Add
> privileges' page, mentioning 'External' as a domain.
> KR,
> Benoît
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/EUSurvey/EUSURVEY/discussions/528#discussioncomment-9466061>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIRDVE74M2ZMYRKAQLBYNC3ZCWMCBAVCNFSM6AAAAABEYDYF7KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TINRWGA3DC>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Giuseppe Chirico
Dipartimento di Fisica
Universita' di Milano-Bicocca
Piazza della scienza 3, 20126
Milano (I)
VAT: IT12621570154
https://www.in2sight.eu
https://https://www.fisica.unimib.it/it/ricerca/biofisica

**Reply by origabe** [reply]
Thank  you for your feedback.
Sorry i forgot to mention that currently the users you would like to give privileges to, should first connect one time to EUSurvey with their EULogin account in order for their address to be known by the system.
It is not linked to your address book. The address book is only to manage the participants to be invited to contribute to your survey.
